### PR TITLE
Add lobby music volume slider

### PR DESCRIPTION
**Pull request by theashtronaut**
## About the PR 

The lobby music is nice. But it's often very loud. I don't want to reduce the master volume, just the lobby volume. It wasn't possible before, but now it is! This only affects the music played in the lobby and during the post round move around part before it kicks to back to the lobby. It will restart the music at the new volume when changed in the lobby, but not during that PostRound part (when you run around and kill each other before it kicks you back to a lobby) as I couldn't figure out how to check for that state easily. It'll still use whatever volume was set so I'm sure that's fine.

I also made all the sliders a consistent size because it looked weird with them all different.

**Screenshots**

https://user-images.githubusercontent.com/112137107/197369863-9996944f-1d1b-4bc2-87cc-69dd031f4ba8.mp4

**Changelog**

:cl: ashtronaut
- add: Added a volume slider for the lobby & round-end music so you can enjoy it without blowing out your eardrums.

### CODE REVIEW

**Comment by theashtronaut** [comment]
test fail whyyyyy

(I think its one of those github moments)

**Comment by theashtronaut** [comment]
image
certified github moment idk why they fail

File: Content.Client/Options/UI/Tabs/AudioTab.xaml [hunk 1]
```diff
@@ -49,7 +49,20 @@
                     <Control MinSize="4 0"/>
                 </BoxContainer>
                 <BoxContainer Orientation="Horizontal" Margin="5 0 0 0">
-                    <Label Text="{Loc 'ui-options-ambience-max-sounds'}" />
+                    <Label Text="{Loc 'ui-options-lobby-volume'}" HorizontalExpand="True" />
+                    <Control MinSize="8 0" />
+                    <Slider Name="LobbyVolumeSlider"
+                            MinValue="0"
+                            MaxValue="300"
```
Comment (rolfero) [inline Content.Client/Options/UI/Tabs/AudioTab.xaml:56]: Do it really need to go all the way up to 300? Currently, master volume goes up to 100%, MIDI (Instrument) volume goes up to 200%, and ambience goes up to 300%.

Comment (theashtronaut) [reply Content.Client/Options/UI/Tabs/AudioTab.xaml:56]: Yeah I just copied the ambience slider. I have no idea why they all have different max values.

Comment (rolfero) [reply Content.Client/Options/UI/Tabs/AudioTab.xaml:56]: I suggest setting it to 200 then maybe

Comment (theashtronaut) [reply Content.Client/Options/UI/Tabs/AudioTab.xaml]: yeah set to 200 to match midi I guess

**Comment by theashtronaut** [comment]
okay i think i resolved the merge conflict successfully. the diff looks mostly right on that file. everything continues to work when i tested locally. the Content.IntegrationTests pass locally so github is just doing another github moment.

**Pull request merged**